packages/forms-web-app/__tests__/unit/controllers/appeals/your-appeals.test.js

```diff
@@ -19,18 +24,72 @@ describe('controllers/appeals/your-appeals', () => {
 
 		jest.resetAllMocks();
 
-		appeals = [{ id: 'appeal123' }];
+		appeal = { id: 'appeal123' };
 		apiClient.getUserByEmailV2.mockImplementation(() => Promise.resolve({ id: '123' }));
-		apiClient.getUserAppealsById.mockImplementation(() => Promise.resolve(appeals));
+		apiClient.getUserAppealsById.mockImplementation(() => Promise.resolve([appeal]));
 		mapToAppellantDashboardDisplayData.mockImplementation((appeal) => appeal);
+		isEligibilityCompleted.mockImplementation(() => false);
+		hasDecisionDate.mockImplementation(() => false);
+		hasFutureDueDate.mockImplementation(() => false);
```

Comment:
I don't think we should mock these - they should be testable.

Reply:
Yeah I agree with this. This looks like integration level testing but it's so heavily mocked it seems spoofed. 

I don't know how feasible this is but I think the tests would be much more robust if you had a set of appeals with expected effects on the page, you `await get` then use [something like this](https://www.npmjs.com/package/@testing-library/jest-dom) to check the DOM you get back has what you expect in it.